### ISSUE TITLE
Unit frames aren't styled in 7.0.3

### ISSUE THREAD

**Issue by Naatan** [cogwerkz/DiabolicUI#4]
![wow-64 2016-08-01 17-43-41](https://cloud.githubusercontent.com/assets/278649/17310716/1bc6e386-5814-11e6-8ffa-9cb2698e081c.png)

offtopic - AWESOME UI! Thanks for creating it.

**Comment by Naatan** [comment]
Note the nameplates also aren't styled. Possibly related?

**Comment by larsnorbergofficial** [comment]
It's not a bug, I simply haven't created those elements yet. My schedule is to "get basic things in place before the Legion launch", so I'll have it all done within 3-4 weeks at the most. :)

**Comment by Naatan** [comment]
Ahh gotcha :) Are you using some sort of third party unit frame for [this screenshot](https://media-curse.cursecdn.com/attachments/191/202/4a285e6c98ce2df9d611d07e68226c08.png) then?

Additionally, would you like bug reports & enhancement requests in the meantime?

**Comment by larsnorbergofficial** [comment]
Bug reports and suggestions are always good. If it's already on my list, no harm done, and if it's something I haven't thought of or aren't aware of, even better! :) 

That screenshot shows the pet frame, and the focus target. I set myself as the focus when I took that screenshot. My plan is to make the 4 party frames in the same style. And raid frames fairly similar, just without the portrait section. :)
